opm/parser/eclipse/EclipseState/Schedule/Network/ExtNetwork.hpp

```diff
@@ -38,14 +38,19 @@ class ExtNetwork {
     ExtNetwork() = default;
     bool active() const;
     bool has_node(const std::string& name) const;
+    bool has_indexed_node_name(std::string name) const;
     void add_node(Node node);
+    void add_indexed_node_name(std::string name);
     void add_branch(Branch branch);
     void drop_branch(const std::string& uptree_node, const std::string& downtree_node);
     const Node& node(const std::string& name) const;
     const Node& root() const;
     std::vector<Branch> downtree_branches(const std::string& node) const;
+    std::vector<const Branch*> branches() const;
     std::optional<Branch> uptree_branch(const std::string& node) const;
     std::vector<std::string> node_names() const;
+    std::vector<std::string> insert_index_nd_names() const;
```

Comment:
Is it necessary to add a new `insert_index_nd_names()` method - I would guess the existing `node_names()` function could be changed to return the new `insert_indexed_node_names()` vector?

Reply:
OK - I have modified the node_names() function.